src/api/notification/notification.ts

```diff
@@ -147,9 +146,9 @@ export class _Notification extends Base implements Notification {
     }
 
     /**
-      Sends a message to the notification.
-      @params {*} message
-      @return {Promise.<void>}
+     * Sends a message to the notification.
+     * @param {*} message
```

Comment:
I have seen `any` used in other parts of the code. we should use one nomenclature.